tags/README.md

```diff
@@ -0,0 +1,15 @@
+# tags firestore collection data model
+
+```
+{
+  'id': string,
+  'date': {
+    "_seconds": number,
```

Comment:
shouldn't this be a ```timestamp```

Reply:
yeah correct. Fixed it.

Reply:
that's weird. Feels a bit in-consistent. Let me discuss this with Vinayak

Reply:
Hi @DashDeipayan The database had some stale data, I created a fresh tag and it is storing as timestamp. 
![image](https://user-images.githubusercontent.com/97341921/227788164-ff241f4a-e1e4-4449-8c05-de50dd750eb7.png)